### PR TITLE
[docs] Using Firebase: refactor the guide with multiple changes

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Fixes ENG-5526

# How

This PR updates the guides page with the following changes:
- removes period at the end of `expo install firebase` command
- fixes a typo (necessary)
- Convert multiple `expo install ...` commands snippets to copy/paste snippets

# Test Plan

The changes have been tested by running docs locally.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/guides/using-firebase.md [hunk 1]
```diff
@@ -18,7 +20,9 @@ If you'd like access to the full suite of native firebase tools, we recommend us
 
 First we need to setup a Firebase Account and create a new project. We will be using the JavaScript SDK provided by Firebase, so pull it into your Expo project.
 
-`expo install firebase`.
+<Terminal cmd={[  
+  '$ expo install firebase'
+]} />
```
Comment (Simek) [inline docs/pages/guides/using-firebase.md:25]:
```suggestion
<Terminal cmd={['$ expo install firebase']} />
```

Let's stick to one line, if we only have one command and only one prop is used, just for the code consistency:
* https://cs.github.com/expo/expo?q=%3CTerminal+cmd%3D%7B%5B

File: docs/pages/guides/using-firebase.md [hunk 2]
```diff
@@ -240,7 +244,9 @@ This sample was borrowed and edited from [this forum post](https://forums.expo.d
 
 In order to record analytics events, the Expo Firebase Core and Analytics packages needs to be installed.
 
-`expo install expo-firebase-analytics`
+<Terminal cmd={[  
+  '$ expo install expo-firebase-analytics'
+]} />
```
Comment (Simek) [inline docs/pages/guides/using-firebase.md:249]: Same as above. 😉

**Review by Simek — COMMENTED** [review]
I have left only a small suggestion, but besides that this LGTM! 👍

File: docs/pages/guides/using-firebase.md (hunk 2 above)
Comment (amandeepmittal) [reply docs/pages/guides/using-firebase.md:247]: Thanks a lot, I was not aware of this. Will keep in mind! I've changed this as per your suggestion.

**Review by Simek — APPROVED** [review]
Thank you for the updates @amandeepmittal! 🙏

**Pull request merged**